### ISSUE TITLE
False positive htmlAttrNotByEscHTML

### ISSUE THREAD

**Issue by kkmuffme** [Automattic/VIP-Coding-Standards#680]
## Bug Description

False positive

## Minimal Code Snippet

```php
for ( $i = 1; $i <= 10; $i++ ) { ?>
	<option value="<?php echo esc_attr( $i ); ?>" <?php echo ( $filter_importance === $i ) ? 'selected' : ''; ?> >
		&gt;=<?php echo esc_html( $i ); ?>
	</option>
<?php } ?>
```

## Error Code

WordPressVIPMinimum.Security.ProperEscapingFunction.htmlAttrNotByEscHTML for `&gt;=<?php echo esc_html( $i ); ?>`

## Environment

Use `php -v` and `composer show` to get versions.

| Question                | Answer
| ------------------------| -------
| PHP version             | 7.4.14
| PHP_CodeSniffer version | 3.6.0
| VIPCS version           | 2.3.1

## Tested Against `master` branch?

- [x] I have verified the issue still exists in the `master` branch of VIPCS.
- [x] I have verified the issue still exists in the `develop` branch of VIPCS.

**Comment by jrfnl** [comment]
This should be fixed by PR #681. Testing appreciated.
